documentation/book/cluster-operator.adoc

```diff
@@ -673,6 +673,117 @@ INFO:: The Cluster Operator doesn't validate the provided configuration. When in
 Kafka Connect cluster might not start or might become unstable. In such cases, the configuration in the `connect-config` field
 should be fixed and the Cluster Operator will roll out the new configuration to all Kafka Connect instances.
 
+===== Kafka Connect S2I deployment
+
+When using {ProductName} together with an {OpenShiftName} cluster, Kafka Connect can be deployed with support for https://docs.openshift.org/3.9/dev_guide/builds/index.html[{OpenShiftName} Builds] and https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)].
```

Comment:
Shouuld we parameterize the version of openshift embedded in the URLs here, to make it easier in the future once the world has moved on and we no longer support 3.9?

Reply:
I do not want to do this, because I know that some of the links are changing over time in the OpenShift docu. And if the version is parametrized, nobody will check that it still works.